create10mEntries.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const entries = 10000000;
+const placeFile = 'dummydata/places.json';
+const reviewFile = 'dummydata/reviews.json';
+
+const createReview = (index) => {
+  const review = {
+    author_name: faker.name.findName(),
+    profile_photo_url: faker.image.imageUrl(),
+    rating: faker.random.number(1, 5),
+    relative_time_description: faker.date.recent(Math.floor(Math.random() * 7)),
+    text: faker.lorem.sentence(),
+    place_id: index,
+  };
+  return review;
+};
+
+const createReviews = (count) => {
+  const reviews = [];
```

Comment:
Should this be const or let?

Reply:
const, editing contents of an array is not reassigning.